install/helm/agones/templates/controller.yaml

```diff
@@ -31,13 +38,13 @@ spec:
       app: {{ template "agones.name" . }}
       release: {{ .Release.Name }}
       heritage: {{ .Release.Service }}
-  replicas: 1
+  replicas: {{ $replicas }}
   strategy:
     type: Recreate
   template:
     metadata:
       annotations:
-{{- if not $featureGates.SplitControllerAndExtensions }}
+{{- if not (gt $replicas 1) }}
```

Comment:
nit: simplify: instead of !(replicas > 1), prefer (replicas == 1) - especially true if we reject <1 above.

Reply:
done